mypy/semanal_main.py

```diff
@@ -335,6 +335,12 @@ def semantic_analyze_target(target: str,
         priority = mypy.build.PRI_LOW
         if priority <= state.priorities.get(dep, priority):
             state.priorities[dep] = priority
+
+    # Clear out some stale data to avoid memory leaks and astmerge
+    # validity check confusion
+    analyzer.statement = None
+    delattr(analyzer, 'cur_mod_node')
```

Comment:
Why not `del analyzer.cur_mod_node`?

Reply:
Because I forgot how to write python apparently